ninetofiver/settings.py

```diff
@@ -275,6 +290,10 @@ def _process_cfg(cls):
     }
     AUTH_LDAP_ALWAYS_UPDATE_USER = True
 
+    # REDMINE 
+    REDMINE_URL = os.getenv('REDMINE_URL')
```

Comment:
No need to use environment variables here. There's perfectly fine code for loading settings here: https://github.com/kalmanolah/925r/blob/master/ninetofiver/settings.py#L38 . There's no need to treat redmine-related variables as special snowflakes, when the database password CAN be loaded from a file.